templates/web/pages/bmviii_home.tmpl

```diff
@@ -0,0 +1,342 @@
+                <div class="columns is-mobile">
+                    <div class="column is-one-third">
+                        <img src={{ static_path "bmvii_why_build.svg" }} class="bmvii-why__img" alt="Gears Turning">
+                        <p class="subtitle is-5 has-text-white has-text-weight-bold" style="font-family: Cabin;">Build Creative Projects</p>
+                        <img src={{ static_path "bmvii_why_resume.svg" }} class="bmvii-why__img" alt="Good resume">
+                        <p class="subtitle is-5 has-text-white has-text-weight-bold" style="font-family: Cabin;">Boost Your Resume</p>
+                    </div>
+                    <div class="column is-one-third">
+                        <img src={{ static_path "bmvii_why_companies.svg" }} class="bmvii-why__img" alt="Formal Attire">
+                        <p class="subtitle is-5 has-text-white has-text-weight-bold" style="font-family: Cabin;">Talk to Company Reps</p>
+                        <img src={{ static_path "bmvii_why_skills.svg" }} class="bmvii-why__img" alt="Brain">
+                        <p class="subtitle is-5 has-text-white has-text-weight-bold" style="font-family: Cabin;">Learn New Skills</p>
+                    </div>
+                    <div class="column is-one-third">
+                        <img src={{ static_path "bmvii_why_memories.svg" }} class="bmvii-why__img" alt="Floppy Disk">
+                        <p class="subtitle is-5 has-text-white has-text-weight-bold" style="font-family: Cabin;">Make Incredible Memories</p>
+                        <img src={{ static_path "bmvii_why_prizes.svg" }} class="bmvii-why__img" alt="Trophy">
+                        <p class="subtitle is-5 has-text-white has-text-weight-bold" style="font-family: Cabin;">Win Cool Prizes</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</section>
+
+<section class="section" style='background: url({{ static_path "bmviii_rings.svg" }}) center center no-repeat, linear-gradient(0, #A7C3FF 50%, #FF8E9E 50%); padding: 6rem 1.5rem;'></section>
+
+<section class="section" id="surfboards" style='position: relative; background-color: #A7C3FF; padding: 0;'>
+    <div class="columns is-centered" style='margin-bottom: 0;'>
+      <div class="column is-half" style='padding-bottom: 0;'>
+        <img src={{ static_path "bmviii_surfboards.svg" }} alt="Surfboards">
+      </div>
+    </div>
+
+    <div style='position: absolute; bottom: -5px; left: -2.5%; width: 105%; font-size: 0;'>
+      <img src={{ static_path "bmviii_sand.svg" }} alt="Sand">
+    </div>
+</section>
+
+<section class="section has-background-image" id="faq-testimonial" style='background-image: url({{ static_path "bmviii_faq_testimonial_background_no_waves.svg" }}); background-size: 1440px 2049px; min-height: 2049px; background-color: #FFE4C3; position: relative;'>
```

Comment:
![Screen Shot 2020-10-30 at 21 08 41](https://user-images.githubusercontent.com/10781907/97767687-b5859780-1aeb-11eb-98ed-c56035380ba5.png)
Is there any way we could get the answer to the question look like a continuation of the question? Like try to replicate the font/padding and especially the color and rounded corners.

Reply:
Color being the grey background?

Reply:
yeah

Reply:
Like this?

![Screen Shot 2020-10-30 at 10 33 08 PM](https://user-images.githubusercontent.com/31838229/97769285-ebcd1200-1aff-11eb-8d71-ef1832b6fe62.png)

Reply:
Maybe get rid of the grey color actual, just for better contrast?

also if you can get it so the transition from the question to the answer is smooth that'd be nice not sure how annoying it'll be to do tho

Reply:
Yeah I agree for the grey color, I like the rounded edges tho

I can double check on that

Reply:
Currently the transition is: max-height 0.2s ease-in-out